api/src/Entity/Period.php

```diff
@@ -85,6 +85,7 @@ class Period extends BaseEntity implements BelongsToCampInterface {
      * @ORM\ManyToOne(targetEntity="Camp", inversedBy="periods")
      * @ORM\JoinColumn(nullable=false)
      */
+    #[Assert\Valid]
```

Comment:
This implies the period can't be updated if the camp is invalid in the DB for some reason. Consider adding `groups: ['Period:delete']`

Reply:
The group 'delete' should be enough i think.

Reply:
It's not, i think only the groups mentioned in the Valid Validation are set during validation of camp...